.github/workflows/optimize_svgs.yml

```diff
@@ -0,0 +1,25 @@
+name: Optimize SVGs
+
+on:
+  schedule:
+    - cron: "0 0 * * 1"
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+  svgs:
+    name: SVGs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Optimize SVGs
+        uses: ericcornelissen/svgo-action@v3
+        id: svgo
+      - name: Commit optimizations
+        uses: stefanzweifel/git-auto-commit-action@v4
+        if: ${{steps.svgo.outputs.DID_OPTIMIZE}}
+        with:
+          commit_message: Optimize ${{steps.svgo.outputs.OPTIMIZED_COUNT}} SVG(s)
```

Comment:
If we can skip commiting if `OPTIMIZED_COUNT` is 0?

Reply:
Updated the version now. Previously it would keep making the same commit each time the workflow ran by trying to optimise an icon which has already been optimise. Now, if there are no SVGs to optimise, there won't be any commit.